relayer/relayer.go

```diff
@@ -173,11 +187,20 @@ func NewRelayer(
 
 // RelayMessage relays a single warp message to the destination chain. Warp message relay requests from the same origin chain are processed serially
 func (r *Relayer) RelayMessage(warpLogInfo *vmtypes.WarpLogInfo, metrics *MessageRelayerMetrics, messageCreator message.Creator) error {
+	// Check that the destination chain ID is supported
+	if len(r.supportedDestinations) > 0 && !r.supportedDestinations.Contains(warpLogInfo.DestinationChainID) {
```

Comment:
Can we create a unit test for this case?

Reply:
I separated this check to its own function and added a unit test.